components/activecampaign/sources/contact-added-to-list/contact-added-to-list.mjs

```diff
@@ -1,22 +1,28 @@
-const activecampaign = require("../../activecampaign.app.js");
-const common = require("../common-webhook.js");
+import activecampaign from "../../activecampaign.app.mjs";
+import constants from "../../common/constants.mjs";
+import common from "../common/webhook.mjs";
 
-module.exports = {
+export default {
   ...common,
   name: "New Contact Added to List",
   key: "activecampaign-contact-added-to-list",
```

Comment:
Since this component uses a webook, the name should have "(Instant)" at the end.
```suggestion
  name: "New Contact Added to List (Instant)",
  key: "activecampaign-contact-added-to-list",
```